### ISSUE TITLE
Accessibility issue: Certain aria roles must contain particular children on /

### ISSUE THREAD

**Issue by juananmora** [juananmora/poc-springboot-sum-integers#1]
## What
An accessibility scan flagged the element `<div class="container" role="menu">` on https://qualitat.solucions.gencat.cat/ because certain aria roles must contain particular children. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.10/aria-required-children?application=playwright.

To fix this, ensure elements with an aria role that require child roles contain them.

Specifically:

Fix any of the following:
-   Element has children which are not allowed: nav[tabindex]

## Acceptance Criteria
- [ ] The specific axe violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific axe violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
